iocBoot/templates/st.cmd.template

```diff
@@ -81,13 +81,13 @@ dbLoadRecords("{{ db.file }}", "{{ db.macros }}")
 {% endfor %}
 cd "$(TOP)"
 
-dbLoadRecords("db/iocAdmin.db", "P={{prefix}},IOC={{prefix}}" )
-dbLoadRecords("db/save_restoreStatus.db", "P={{prefix}},IOC={{name}}" )
+dbLoadRecords("db/iocAdmin.db", "P={{prefix}}{{delim}},IOC={{prefix}}{{delim}}" )
+dbLoadRecords("db/save_restoreStatus.db", "P={{prefix}}{{delim}},IOC={{name}}" )
 
 # Setup autosave
 set_savefile_path( "$(IOC_DATA)/$(IOC)/autosave" )
 set_requestfile_path( "$(TOP)/autosave" )
-save_restoreSet_status_prefix( "{{prefix}}:" )
+save_restoreSet_status_prefix( "{{prefix}}{{delim}}" )
```

Comment:
What does `save_restoreSet_status_prefix` do again?

Reply:
Quoting [Tim Mooney](https://epics.anl.gov/tech-talk/2011/msg00496.php):
```
Autosave is partly a channel-access client, and the save_restoreStatus.db
database is sort of its user interface.  The dbLoadRecords() command
creates PVs for autosave to use, but that doesn't help autosave find out
what their names are, because autosave doesn't have access to the macro
values supplied to the dbLoadRecords command.
```

Reply:
Thanks :+1:

Reply:
I think 99% of my confusion though was brain derp on the definition of a delimiter